PendingReleaseNotes

```diff
@@ -131,3 +131,7 @@
   The above options can be set with::
 
     ceph config set global <option> <value>
+
+* Bucket notifications now support Kafka endpoints. This require librdkafka of version 0.9.2 and up.
+  Note that Ubuntu 16.04.6 LTS (Xenial Xerus) has an older version of libradkafka, and would
```

Comment:
is `libradkafka` a typo? (The previous line has `librdkafka`.)

Reply:
yes, should be "librdkafka"

Reply:
fixed